infrastructure/crypto/src/main/java/tech/pegasys/teku/infrastructure/crypto/MessageDigestFactory.java

```diff
@@ -25,17 +25,28 @@ public class MessageDigestFactory {
   private static final Logger LOG = LogManager.getLogger();
 
   public static final String SHA_256 = "SHA-256";
-  private static final Provider SECURITY_PROVIDER = selectSecurityProvider();
+  public static final String KECCAK_256 = "KECCAK-256";
+  private static final Provider SHA_256_SECURITY_PROVIDER = selectSha256SecurityProvider();
+  private static final Provider KECCAK_256_SECURITY_PROVIDER = new BouncyCastleProvider();
```

Comment:
Probably worth having a constant `BOUNCY_CASTLE_PROVIDER` since we'll always need it now, then just use that existing instance here and in `selectSha256SecurityProvider`.